.travis.yml

```diff
@@ -22,7 +22,8 @@ script:
     - export JULIA_PROJECT=""
     - julia --project -e 'using Pkg; Pkg.build(); Pkg.test();'
     - julia -e 'using Pkg; Pkg.develop(PackageSpec(path="."));
-                           Pkg.add(PackageSpec(url="https://github.com/timholy/Revise.jl"));
+                           Pkg.add(PackageSpec(url="https://github.com/JuliaDebug/JuliaInterpreter.jl", rev="ct1"));
+                           Pkg.add(PackageSpec(url="https://github.com/timholy/Revise.jl", rev="revise3"));
```

Comment:
do these need to be removed after some releases are made?

Reply:
Maybe even before merging. These are not needed for package functionality, but this runs the Revise tests and the circular dependency means we need a compatible version of Revise.